Corrected text to include Validated Content (#495)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/assemblies/hub/assembly-faq.adoc

If you have any questions regarding the items below, please email ansiblepartners@redhat.com.

== Why certify Ansible collections?

The Ansible certification program enables a shared statement of support for {certifiedCon} between Red Hat and the ecosystem partner. 
An end customer, experiencing trouble with Ansible and certified partner content, can open a support ticket (request for information, a problem, etc.) with Red Hat and expect the ticket will be resolved by Red Hat and the ecosystem partner. 

Red Hat offers go-to-market benefits for Certified Partners to grow market awareness, demand generation and collaborative selling.

Red Hat {CertifiedName} are distributed through {HubNameMain} (subscription required), a centralized repository for jointly supported Ansible Content. 
As a certified partner, publishing Collections to {HubNameMain} provides end customers the power to manage how trusted automation content is used in their production environment with a well known support life cycle.

For more information about getting started with certifying a solution, please refer to https://www.ansible.com/partners for details.

> //As a partner, publishing Collections to the Automation Hub gives your end customer organizations the power to manage how trusted automation content is used in the customer environment.

Reviewer: Delete this line, as it's included further up

Once users are added as administrators of the namespace, then additional administrators can be added by the self-serve process.

== Are there any restrictions for Galaxy namespace naming?

Collection namespaces must follow python module name convention. 
This means collections should have short, all lowercase names. 
Underscores may be used in the collection name if it improves readability.

== Are there any naming recommendations for collection naming?

A general suggestion is to create a collection with `company_name.product` format. 
This way multiple products may have different collections under the company namespace.

== How do I get a namespace on {HubNameMain}?

By default namespaces used on {Galaxy} are also utilized on {HubNameMain} by the Ansible partner team. 
For any queries and clarifications please contact ansiblepartners@redhat.com.

== How do I run sanity tests on my collection?

Ansible sanity tests are made up of scripts and tools used to perform static code analysis. 
The primary purpose of these tests is to enforce Ansible coding standards and requirements. 

> Ansible collections must be in a very specific path, such as the following:

Reviewer: ```suggestion
Ansible collections must be in a specific path, such as the following:
```

[options="nowrap" subs="=quotes, attributes"]
----
{...}/ansible_collections/{namespace}/{collection}/
----

You must ensure your collection is in that specific path, with an empty directory named `ansible_collections`, then a directory for the namespace, and finally a directory for the collection itself.

== Does Galaxy house the source code for my Collection?

No, Galaxy does not house the source for the collections. 

> The actual collection source must be housed outside of {Galaxy}, for example GitHub. 

Reviewer: ```suggestion
The actual collection source must be housed outside of {Galaxy}, for example in GitHub. 
```

{Galaxy} contains the collection build tarball to publish the collection. 
You can include the link to the source for community users in the `galaxy.yml` file contained in the collection. 

> This provides information to users of where they should go if they want to contribute to the collection or even file issues against it.

Reviewer: ```suggestion
This shows users where they should go if they want to contribute to the collection or even file issues against it.
```

== How does the joint support agreement on certified collections work?

If a customer raises an issue with the Red Hat support team about on a certified collection, Red Hat support will triage the issue and check if the problem exists within Ansible or Ansible usage and they also check if the issue is with a certified collection. 
If there is a problem with the certified collection, support teams transfer the issue to the vendor owner of the certified collection through an agreed upon tool such as TSANet.

== How to migrate existing modules/plugins to collections?

> Follow the guide found here <link required TBD> to start developing a collection. 

Author: They haven't supplied it!
